RIOT-OS/RIOT
# pkg/semtech-loramac: move region CFLAGS definitions from applications to pkg

### Contribution description

This simplify the Makefile of each lorawan applications.




### Testing procedure

Build and flash `examples/lorawan` and `tests/pkg_semtech-loramac` applications and verify that they work as expected (on region EU868, the default)




### Issues/PRs references

Required by #8855
